packages/documentation/.storybook/preview.js

```diff
@@ -49,4 +68,9 @@ export const parameters = {
       date: /Date$/,
     },
   },
+  darkMode: {
+    darkClass: 'bg-primary',
```

Comment:
```suggestion
    darkClass: 'bg-dark',
```
In the moment it is the same color, but in the future this could change and then it will make more sense to use the `bg-dark` class than the `bg-primary`.